backend/src/recommitment/cron.service.ts

```diff
@@ -16,39 +16,90 @@ export class CronService {
   }
 
   /**
-   * Initial Job Scheduled
+   * Initial Job Scheduled for 6am on Jan 13th or otherwise specified
    */
-  @Cron(`0 0 13 1 *`, {
+  @Cron(process.env.START_DATE ?? '0 6 13 1 *', {
     name: 'initial_recommitment',
     timeZone: 'America/Vancouver',
   })
   async initialRecommitment() {
-    const recurringNotificationsJob = new CronJob('0 0 * * 1', async () => {
-      await this.recommitmentService.handleSendAutomatedReminders();
-    });
+    // Start Recommitment. If not in production, use test email and do not send emails to members
+    process.env.ENV === 'production'
```

Comment:
Is `ENV` the one that's production across all deployed environments?

Reply:
No that one is "NODE_ENV". This one uses dev/test in the those environments.